### ISSUE TITLE
"We don't own this object" error during annotation creation on mobile app

### ISSUE THREAD

**Issue by sbutkovi** [eternagame/EternaJS#627]
Specifically, the message seems to appear when navigating the radio buttons for "Puzzle" and "Solution" - see screenshots below:

![pic1](https://user-images.githubusercontent.com/35237235/160923919-47d61fe3-2151-41bb-8837-7fd2ddb013f9.PNG)

![pic2](https://user-images.githubusercontent.com/35237235/160923935-20dbb934-bf8d-4e0b-b669-33176953298b.png)

**Comment by luxaritas** [comment]
May or may not be touch-specific. Having a hard time reproducing in the browser mobile mode, but will need to play around a bit more.

As a side note, that popup/modal looks to be getting cut off or running into some other margin/padding issues

**Comment by sbutkovi** [comment]
There aren't any more "We don't own this object" errors, but there is still some odd behavior.

In this screenshot, I was having a lot of trouble trying to switch the radio bubble choice from Solution to Puzzle - not entirely sure if that was due to it overlapping the toolbar (from the huge keyboard pop-up) or a different issue. I somehow got "around the interface" enough to click the button to make the annotation, but I can't remember what I did besides that it took some effort:

![pic1](https://user-images.githubusercontent.com/35237235/162646362-7772050d-4742-4267-845b-207d3577b0d0.PNG)

However, upon creating the annotation, I was softlocked into the following puzzle state. I could not seem to interact with any buttons or change the puzzle view in any way, but the app was still running (the bubbles were still moving). Seems fixable only by reloading the game.

![pic2](https://user-images.githubusercontent.com/35237235/162646365-6c133cab-1bd2-4e4e-a210-d8453ad4e21b.png)
